package.json

```
@@ -7,7 +7,7 @@
     "@typescript-eslint/experimental-utils": "^4.0.0",
     "common-tags": "^1.8.0",
     "decamelize": "^5.0.0",
-    "eslint-etc": "^4.0.0",
+    "eslint-etc": "^4.0.2",
```

Thanks. We should actually bump this further - to `4.0.4` - as that's what's required for the tests to work properly. I mean, `4.0.2` should be fine for dev installs because of the lock file, but we might as well set it to `4.0.4`.
```suggestion
    "eslint-etc": "^4.0.4",
```